accname/name/comp_name_from_content_menuitem.html

```diff
@@ -0,0 +1,107 @@
+    </li>
+    <li>
+      <a role="menuitem" href="#" aria-labelledby="aria-labelledby-image" data-expectedlabel="image" data-testname="anchor name from content with menuitem aria-labelledby override" class="ex">
+        <img src="#" id="aria-labelledby-image" alt="image">
+      </a>
+    </li>
+    <li>
+      <a role="menuitem" href="#" data-expectedlabel="image" data-testname="anchor name from content with menuitem nested img with alt override" class="ex">
+        <img src="#"  alt="image">
+      </a>
+    </li>
+  </ul>
+
+  <h1>Named from Content for Button Menuitems</h1>
+  <ul role="menu">
+    <li>
+      <button role="menuitem" data-expectedlabel="Menuitem button named from content" data-testname="button name from content with menuitem" class="ex">Menuitem button named from content</button>
+    </li>
+    <li>
+      <button role="menuitem" data-expectedlabel="" data-testname="button name from content with menuitem empty string" class="ex"></button>
+    </li>
+  </ul>
+
+  <h2>Menuitem Button Author Overrides</h2>
+  <ul role="menu">
+    <li>
+      <button role="menuitem" aria-label="Menuitem button named by aria label" data-expectedlabel="Menuitem button named by aria label" data-testname="button name from content with menuitem aria-label override" class="ex">Menuitem button named from content</button>
+    </li>
+    <li>
+      <button role="menuitem" aria-labelledby="button-span" data-expectedlabel="Button Span" data-testname="button name from content with menuitem empty string" class="ex"><span id="button-span">Button Span</span></span></button>
+    </li>
+    <li>
+      <button role="menuitem" data-expectedlabel="icon" data-testname="button name from content with menuitem with nested image alt" class="ex"><img src="#" alt="icon"></button>
+    </li>
+  </ul>
+
+  <h1>Named from Content for Table Heading Menuitems</h1>
+  <table role="menu">
+    <tr role="menuitem" data-expectedlabel="Table row named from content" data-testname="table row name from content with menuitem" class="ex">
+      <th>Table row named from content</th>
+    </tr>
+    <tr role="menuitem" data-expectedlabel="" data-testname="table row name from content with menuitem empty string" class="ex">
+      <td></td>
+    </tr>
+  </table>
+
+  <h2>Menuitem Table Headings Author Overrides</h2>
+  <table role="menu">
+    <tr role="menuitem" aria-label="Menuitem table row named by aria label" data-expectedlabel="Menuitem table row named by aria label" data-testname="table row name from content with menuitem named by aria-label " class="ex">
+      <th>Table row named from content</th>
+    </tr>
+    <tr role="menuitem" data-expectedlabel="data-icon" data-testname="Menuitem table row named by image alt" class="ex">
+      <td><img src="" alt="data-icon"></td>
+    </tr>
+  </table>
+
+<script>
+AriaUtils.verifyLabelsBySelector(".ex");
```

Comment:
FYI we've started trying to update this original pattern with more specific class names, like .ex-label and .ex-role, in case later we decide to perform both role and label tests on a single element.